### PR TITLE
typo_fix_hn_trans

### PR DESCRIPTION
**Pull request by raj-maurya**
I have fixed some hindi translation typos.

Please review.

Thank you

### CODE REVIEW

**Comment by jwasham** [comment]
Should I pull now or later?

**Comment by MrSourabh** [comment]
@jwasham I am working on next chunk, i think you can merge at once it is complete. Thanks and wish you happy new year

**Comment by jwasham** [comment]
When you're ready for me to pull, let me know.

**Comment by jwasham** [comment]
I could pull this, but it needs de-conflicting.

File: README-hn.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ memory" का एरर न दे, और तब मुजे कोई वै
 
 ## कभीबी आप चालक नहीं हो ऐसा ना सोचो
```
Comment (prabodhmeshram) [inline README-hn.md:151]:
```suggestion
## आप चालक नहीं हो, ऐसा कभी भी ना सोचे
```

File: README-hn.md [hunk 2]
```diff
@@ -240,9 +240,9 @@ It is free to do so, but sometimes the classes are not in session so you have to
 
 ### १. आपसे यह सब याद नहीं होगा
 
-मैंने घंटो वीडिय के विडियो देखे और टिप्पणिया लिखी, और महीनो बाद मुजे कुछ याद नहीं रहा. सबकी समीक्षा करने के लिए मैंने 3 दिन मेरी तिप्पनिओयो और flashcards बनाने में बितायें (नीचे देखें).
+मैंने घंटो वीडिय के विडियो देखे और टिप्पणिया लिखी, और महीनो बाद मुजे कुछ याद नहीं रहा. सबकी समीक्षा करने के लिए मैंने 3 दिन मेरी तिप्पनियो और flashcards बनाने में बितायें (नीचे देखें).
```
Comment (prabodhmeshram) [inline README-hn.md:243]: मैंने घंटो घंटो विडियो देखे और टिप्पणिया लिखी, और महीनो बाद मुझे कुछ याद नहीं रहा. सबकी समीक्षा करने के लिए मैंने ३ दिन मेरी तिप्पनियो और फ़्लैशकार्ड बनाने में बितायें (नीचे देखें).

**Comment by jwasham** [comment]
Please fix conflicts and make edits and I can merge it in.

**Comment by prabodhmeshram** [comment]
@jwasham I cannot fix them as the contributor is someone else, I've just added my bits of comment. Although this change goes back to 2016-17, I doubt these changes would be relevant anymore. A new fresh MR would be good idea in that case, rather resolving the conflicts.

**Comment by kevonokia** [comment]
Ok use thus no+254795206368

**Review by saurabh-mhaisekar — APPROVED** [review]

**Comment by RAJUKUMAR-LAB** [comment]
Nahi massage mat Karo

**Comment by jwasham** [comment]
Will close this due to age and the small number of changes.

**Pull request closed without merging**